app/posts/govuk-eleventy-plugin.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Write documentation using Markdown and publish it using GOV.UK styles
+description: The GOV.UK Eleventy Plugin makes it easy to start writing documentation rather than spend time building a website for it.
+date: 2022-04-04
+tags: post
+layout: post
+author:
+  name: Paul Lloyd
+  url: https://github.com/paulrobertlloyd
+image:
+  src: /assets/posts/govuk-eleventy-plugin/govuk-eleventy-plugin.png
+  alt: The homepage illustration for the GOV.UK Eleventy Plugin.
+---
+
+Back in 2018, the Becoming a teacher team at the Department for Education wanted to [document the design evolution of the services they were building](https://design-history.herokuapp.com/keeping-a-design-history/). A small site was built using the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/docs), but this soon became difficult to manage as more people joined the team and wanted to write posts.
+
+Looking for alternatives, the team choose [Eleventy](https://11ty.dev), a simple static site generator developed by Zach Leatherman[^1]. Like the Prototype Kit, it uses Node.js, yet is designed around organising and publishing content-focused websites.
```

Comment:
Minor typo:

```suggestion
Looking for alternatives, the team chose [Eleventy](https://11ty.dev), a simple static site generator developed by Zach Leatherman[^1]. Like the Prototype Kit, it uses Node.js, yet is designed around organising and publishing content-focused websites.
```

I also wonder whether you could briefly explain what a “static site generator” is here, for people not familiar with the term?